pages/ksphere/konvoy/partner-solutions/jenkins/index.md

```diff
@@ -90,19 +90,19 @@ Your Jenkins instance will deploy.
         defaultCrumbIssuer:
           enabled: false
           proxyCompatability: false
-    prometheus:
-      enabled: true
-      serviceMonitorNamespace: "kubeaddons"
-      serviceMonitorAdditionalLabels:
-      app: jenkins
-      release: prometheus-kubeaddons
-    serviceType: "LoadBalancer"
-    jenkinsUriPrefix: "/jenkins"
-    ingress:
-      enabled: true
-      path: /jenkins
-      annotations:
-        kubernetes.io/ingress.class: traefik
+      prometheus:
+        enabled: true
+        serviceMonitorNamespace: "kubeaddons"
+        serviceMonitorAdditionalLabels:
+        app: jenkins
+        release: prometheus-kubeaddons
```

Comment:
these two labels don't need to be indented?

Reply:
🤦 thanks a lot @samvantran !